### PR TITLE
Create a file for githubteacher raleigh

### LINKED ISSUE #10: Githubteacher's recommendations for Raleigh, NC
## Adding Raleigh, NC

I would like to add a file for Raleigh, NC to this repository.

I will follow the GitHub flow:

- [x] Create a branch
- [x] Add my file
- [x] Commit my changes
- [x] Open a pull request 
- [ ] Collaborate with my friends
- [ ] Get a :+1:
- [ ] Merge my Pull Request

### PR DESCRIPTION
**Pull request by githubteacher**
This work will close #10 

## Things left to do

- [x] Add a list of places to eat
- [x] Add a list of fun things to do

### CODE REVIEW

File: githubteacher-raleigh.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Recommendations in Raleigh, NC
+
+### Places to eat
```
Comment (githubteacher) [inline githubteacher-raleigh.md]: What are your favorite types of foods? @WhatWouldHoneyBadgerDo

Comment (WhatWouldHoneyBadgerDo) [inline githubteacher-raleigh.md]: Carolina BBQ!

File: githubteacher-raleigh.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Recommendations in Raleigh, NC
+
```
Comment (jeffreykey) [inline githubteacher-raleigh.md]: Looking forward to hearing about Raleigh!

File: githubteacher-raleigh.md (hunk 1 above)
Comment (jeffreykey) [inline githubteacher-raleigh.md]: BBQ sounds delicious!

**Pull request merged**